phpunit.xml.dist

```diff
@@ -21,5 +21,14 @@
 
     <logging>
         <log type="coverage-clover" target="build/reports/phpunit.xml" />
+        <log type="coverage-html"
+            title="php-assumptions"
+            target="build/coverage"
+            charset="UTF-8"
+            yui="true"
+            highlight="true"
+            lowUpperBound="35"
+            highLowerBound="70" />
+        <log type="coverage-clover" target="build/logs/clover.xml"/>
```

Comment:
`coverage-clover` is already in there and perhaps it's overriding the previous which makes it so it can't send the report to scrutinizer.